Kernel/CMakeLists.txt

```diff
@@ -273,46 +273,17 @@ set(SOURCES
     ${CRYPTO_SOURCES}
     ${C_SOURCES}
 )
-set(KERNEL_FSANITIZE_FLAGS
-    "shift"
-    "shift-exponent"
-    "shift-base"
-    "integer-divide-by-zero"
-    "unreachable"
-    "vla-bound"
-    "return"
-    "signed-integer-overflow"
-    "bounds"
-    "bounds-strict"
-    "object-size"
-    "float-divide-by-zero"
-    "float-cast-overflow"
-    "nonnull-attribute"
-    "returns-nonnull-attribute"
-    "bool"
-    "enum"
-    "vptr"
-    "builtin"
-    "alignment"
-    "null"
-    "pointer-overflow"
-    )
-
-if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    foreach(flag IN LISTS KERNEL_FSANITIZE_FLAGS)
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=${flag}")
-    endforeach()
-endif()
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unknown-warning-option")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pie -fPIE -fno-rtti -ffreestanding -fbuiltin")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mno-80387 -mno-mmx -mno-sse -mno-sse2")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-asynchronous-unwind-tables")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fstack-protector-strong")
 if (NOT ${CMAKE_HOST_SYSTEM_NAME} MATCHES SerenityOS)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -nostdlib -nostdinc -nostdinc++")
```

Comment:
Why this indentation change?

Reply:
I don't know, but I'll fix that probably today

Reply:
Should be fixed now,  
Additionally I've readded the compiler check for fsanitize